### ISSUE TITLE
[FEATURE_REQUEST] [WORKING EXAMPLE READY] Extension-Auto-Send

### ISSUE THREAD

**Issue by randoentity** [SillyTavern/SillyTavern#1617]
### Have you searched for similar requests?

None

### Is your feature request related to a problem? If so, please describe.

tl;dr: I want hands-free STT on Android.

While I'm going about my day I like to have SillyTavern open on my phone and I want to interact with it naturally. Ideally I want to integrate it with my home automation system some day as well. STT streaming isn't supported out of the box on commonly owned pocketable devices, and I don't want to touch my phone to send messages. There are some neat open source IME apps using Vosk and Whisper (Sayboard, Futo respectively), of which the former needs no tactile input to enter text once invoked. The only piece missing for me was a way to actually send my message, which Streaming STT does support, but regular text entry (via voice IME in this case) does not.

### Describe the solution you'd like

Auto send a message once the user has:
- Entered some text into the message box
- Is idle for a certain duration
- Has the Auto Send feature enabled

### Describe alternatives you've considered

- SBC with audio input
- Putting my laptop in my pocket
- Bluetooth extender
- Microphones everywhere
- ~~Alexa~~

### Additional context

This works (for me, at the time of writing): https://github.com/randoentity/Extension-Auto-Send and is a copy-paste of https://github.com/SillyTavern/Extension-Idle/ with some minor tweaks.

Also: I looked through the docs and the UI but other than the group chat and the streaming auto sending, I could not find something that fit the bill.

### Priority

Low (Nice-to-have)

### Is this something you would be keen to implement?

Yes!

**Comment by randoentity** [comment]
The next thing I'll want to add is a way to issue commands, especially `/del 2` and `/autosend` (to disable it). It will be nice to get hotwords working upstream (in the IME), but it not being common tells me it might not be easy. 
Also as an alternative to mapping "replace words" here (e.g. "forward slash" to "/") it could be done upstream (relevant: https://github.com/ElishaAz/Sayboard/issues/18).

Another alternative to this issue: with the vosk/whisper stt extension: implement a similar timeout as here. Looking at the docs this might already work in some setups ("Recording may stop automatically if no voice is detected.") but I have to manually tap the mic button.

Also I thought about trying a browser which supports STT, but I use Firefox.
